README.md

```diff
@@ -1,4 +1,5 @@
-[![Reviewed by Hound](https://img.shields.io/badge/ESLint%20Reviewed%20by%20-HoundCI-d16ef5)](https://houndci.com)
```

Comment:
Why was this removed?

Reply:
I made a PR to master before, but when I switched base to develop, it conflicted with Chima's code because he already merged to that branch, that's why I removed it.

Reply:
That is a very wrong way to resolve merge conflicts. If this was a new feature, It is definitely sure that you would be introducing a bug to the application.

Reply:
Always make enquiries (or be absolutely sure) before deleting any line of code. In any event, I have reverted the changes pertaining to the HoundCI integration. Be sure to rebase from develop and correctly resolve you merge conflicts

Reply:
Ok thank you noted @eguonoefe